ggshield/scan/repo.py

```diff
@@ -110,7 +112,7 @@ def scan_commits_content(
 
 
 def get_commits_by_batch(
-    commit_list: Iterable[Commit],
+    commit_list: List[Commit],
```

Comment:
Out of curiosity: do we need this change? Taking an Iterable is nicer.

Reply:
No we don't, this was an unwanted change left after my experiments.